src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java

```diff
@@ -69,7 +69,7 @@ public double yawAdjustment() {
     }
 
     public double getVoltage() {
-        // return 12;
-        return Robot.getInstance().PDP.getVoltage();
+         return 12;
+         //kinda want pdp for better accuracy but yall removed it
```

Comment:
The latest version of my PR reimplements this in the hardware class, reference that for voltage